_posts/2016-9-5-this-week-in-rust-docs-20.md

```diff
@@ -0,0 +1,87 @@
+
+# Latest news (getting a bit old but still gold!)
+
+Please take a look [to the next rust doc days planning reminder](https://users.rust-lang.org/t/reminder-planning-the-next-rust-doc-days/6901).
+
+The topic to propose crates for the Rust Doc Days is still open and waiting for contributions [here](https://users.rust-lang.org/t/call-for-proposals-for-next-rust-doc-days-crates/6685). Please take a look!
+
+# Current opened issues
+
+For now, here are the three big issues for Rust documentation:
+
+* [Error code list which need to be updated to new format](https://github.com/rust-lang/rust/issues/35233)
+* [The Standard Library Documentation Checklist](https://github.com/rust-lang/rust/issues/29329)
+* [Add error explanations for all error codes](https://github.com/rust-lang/rust/issues/32777)
+
+They all need help to move forward so any contribution is very welcome!
+
+There are currently around 70 other documentation issues opened. Look for [A-docs](https://github.com/rust-lang/rust/issues?q=is%3Aopen+is%3Aissue+label%3AA-docs) tagged issues on GitHub!
+
+# Call for participation
+
+There's now a call for participation to [display all methods of a type, even those from implicit traits in rustdoc](https://github.com/rust-lang/rust/issues/33772). This is a great way to help users find everything that a type can do. Any help on it would be very appreciated!
+
+# Waiting for merge
+
+* [@matthew-piziak](https://github.com/matthew-piziak) added [links to interesting items in `std::ptr` documentation](https://github.com/rust-lang/rust/pull/35880) and [refactored range examples](https://github.com/rust-lang/rust/pull/35759)
+* [@frewsxcv](https://github.com/frewsxcv) indicated [where `core::result::IntoIter` is created](https://github.com/rust-lang/rust/pull/35845)
+* [@steveklabnik](https://github.com/steveklabnik) made it clear that [reference isn't normative](https://github.com/rust-lang/rust/pull/35102)
+* [@ollie27](https://github.com/ollie27) fixed [associated consts in search results](https://github.com/rust-lang/rust/pull/36078)
+* [@GuillaumeGomez](https://github.com/GuillaumeGomez) fixed [formatting generation for rustdoc code examples](https://github.com/rust-lang/rust/pull/35012) and [added missing urls](https://github.com/rust-lang/rust/pull/36243)
+* [@Cobrand](Cobrand) added [mention of `make tidy`](https://github.com/rust-lang/rust/pull/36241)
+* [@c4rlo](https://github.com/c4rlo) fixed [a "\\" in a table heading to be "/"](https://github.com/rust-lang/rust/pull/36204)
+* [@apasel422](https://github.com/apasel422) cleaned up [thread-local storage docs](https://github.com/rust-lang/rust/pull/36263)
+
+
+# Recent doc contributions
+
+This week (too), I'll split between the [new error code format](https://github.com/rust-lang/rust/issues/35233) contributions and the others. Let's start by the first one:
```

Comment:
Huh. I believe it  _is_ a comma. Do you have a speck of dust on your screen, perhaps?